client/my-sites/plan-features/item.jsx

```diff
@@ -9,31 +9,48 @@ import classNames from 'classnames';
  */
 import Gridicon from 'components/gridicon';
 import InfoPopover from 'components/info-popover';
+import { abtest } from 'lib/abtest';
 
-export default function PlanFeaturesItem( { description, children, className } ) {
-	const renderTipinfo = () => {
-		if ( ! description ) {
-			return null;
-		}
+//export default function PlanFeaturesItem( { description, children, className } ) {
+export default React.createClass( {
+
+	displayName: 'PlanFeaturesItem',
 
+	getInitialState() {
```

Comment:
@artpi did you run into any trouble with binding, that made you use `React.createClass` over an es6 class?